content/deploy/tutorials/docker.md

```diff
@@ -16,28 +16,25 @@ Wherever you decide to deploy your app, you will first need to containerize it.
 
 ## Prerequisites
 
-1. [Install Docker Engine](#install-docker-engine)
+1. [Install Docker](#install-docker)
 2. [Check network port accessibility](#check-network-port-accessibility)
 
-### Install Docker Engine
+### Install Docker
 
-If you haven't already done so, install [Docker](https://docs.docker.com/engine/install/#server) on your server. Docker provides `.deb` and `.rpm` packages from many Linux distributions, including:
-
-- [Debian](https://docs.docker.com/engine/install/debian/)
-- [Ubuntu](https://docs.docker.com/engine/install/ubuntu/)
```

Comment:
[The official documentation](https://docs.docker.com/desktop/setup/install/linux/#supported-platforms) talks about this, with expanded information. Linking there instead.